dev/bots/test.dart

```diff
@@ -423,16 +427,16 @@ Future<void> _runFrameworkTests() async {
     final Directory flutterDriverPackageTestDirectory = Directory(path.join(flutterDriverPackageDirectory.path, 'test'));
 
     final List<String> flutterDriverTests = flutterDriverPackageTestDirectory
```

Comment:
This seems far too involved for what it is doing. I would instead place the test you want to skip in an entirely separate directory like:

test_driver/test/
   /real_tests
   /web_tests